template/src/components/views/NotFound.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+  AppBar,
+  Button,
+  Container,
+  CssBaseline,
+  Grid,
+  Link,
+  Tab,
+  Tabs,
+  Toolbar,
+  Typography,
+} from '@material-ui/core';
+import { NavLink } from 'react-router-dom';
+import UserMenu from './UserMenu';
+import background404 from 'assets/img/404.svg';
+
+const useStyles = makeStyles((theme) => ({
+  grid: {
+    width: '100vw',
+    height: '100vh',
+    overflow: 'hidden',
+  },
+  containerWrapper: {
+    height: 'calc(100% - 56px)',
+  },
+  contentWrapper: {
+    height: '100%',
+    backgroundImage: `url("${background404}")`,
+    backgroundPosition: 'bottom',
+    backgroundRepeat: 'no-repeat',
+    backgroundSize: 'contain',
+  },
+  actionWrapper: {
+    marginTop: '24px',
+  },
+}));
+
+export default function NotFound() {
+  const classes = useStyles();
+
+  return (
+    <Grid container direction='column' className={classes.grid}>
+      <CssBaseline />
+      <AppBar position='static' className={classes.navBar}>
```

Comment:
We're replicating the AppBar and it will force users to maintain two AppBar.

For example, if we add a new section you need to add in Home and here.

Reply:
Yes, I'm refactor in the next PR